Controllers/CHumidityController.cpp

```diff
@@ -8,6 +8,7 @@
 #include "CHumidityController.h"
 #include <stdio.h>
 #include "CBME280.h"
+#include "CHumidifier.h"
```

Comment:
No, I included that while I was previously using the CHumidifier class to do the humidifier abstraction. I forgot to remove it since it's no longer being used.